meta/documents/changelog_en.md

```diff
@@ -1,46 +1,51 @@
-﻿## v1.0.0 (2019-03-06)
-- Release of plugin with following functions: hourly order import, hourly inventory update, automated shipping label generation
+﻿# Release notes for Wayfair
 
-## v1.0.1 (2019-04-03)
-- Added functionality of matching incoming orders with plentymarkets item using EAN and marketplace-specific SKU.
-- Updated User Guide.
+## v1.1.2 (2020-05-15)
+- Add support for paginated inventory synchronization
+- Fixed downloading empty documents to plentymarkets
+- Increased logging
+- Increased telemetry
 
-## v1.0.2 (2019-07-09)
-- Fix wrong URLs with documentation
+## v1.1.1 (2020-01-08)
+- Fixed inventory mapping using SKU
 
-## v1.0.3 (2019-07-17)
-- Only pull "open" orders
+## v1.1.0 (2019-12-26)
+- Fixed inventory mapping using SKU
+- Fixed wrong address field when creating customer address
+- Fixed not sending tracking number.
 
-## v1.0.4 (2019-08-13)
-- Updated Wayfair address in the plugin
+## v1.0.10 (2019-11-19)
+- Fixed inventory mapping using EAN or market-specific SKU option
+- Fixed ASN missing Zip code in billing address
 
-## v1.0.5 (2019-08-26)
-- Allow full inventory update for live applications
-- Clean up negative values in inventory feed
+## v1.0.9 (2019-11-08)
+- Critical fix for full inventory update
+
+## v1.0.7 (2019-10-23)
+- Add packing slip to the order 
+- Add the option of choosing to send all items inventory data
 
 ## v1.0.6 (2019-09-06)
 - Fix issue with large full inventory update
 - Fix setting wrong amount for orders with more than 1 item
 - Update plugin intro page
 
-## v1.0.7 (2019-10-23)
-- Add packing slip to the order 
-- Add the option of choosing to send all items inventory data
+## v1.0.5 (2019-08-26)
```

Comment:
Why was this date modified?

Reply:
this is just what the diff looks like after I flipped this file upside down to have latest releases go on the top.